website/docs/d/cis_dns_record.html.markdown

```diff
@@ -0,0 +1,46 @@
+---
+
+layout: "ibm"
+page_title: "IBM : Cloud Internet Service DNS Record"
+sidebar_current: "docs-ibm-datasources-cis-dns-record"
+description: |-
+Manages IBM Cloud Internet Service DNS Record.
+
+---
+
+# ibm_network_cis_dns_record
+
+Import the details of an existing IBM Cloud Internet Service domain name service record as a read-only data source. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+
+data "ibm_network_cis_dns_records" "test" {
+  cis_id    = var.cis_crn
+  domain_id = var.zone_id
+}
+
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+- `cis_id` - (Required, string) The resource cis id of the CIS on which zones were created.
+- `domain_id` - (Required, string) The resource domain id of the DNS on which zones were created.
+
+## Attribute Reference
```

Comment:
only computed fields should be referred in attribute reference
please follow the schema structure in attribute reference, for example:
*`cis_dns_records `
      *`id `
      * `record_id`

Reply:
done